components/director/hack/plugins/scopesdecorator/plugin.go

```diff
@@ -21,7 +23,7 @@ const (
 
 var _ plugin.ConfigMutator = &scopesDecoratorPlugin{}
 
-func NewPlugin(schemaFileName string) *scopesDecoratorPlugin {
+func NewScopesDecoratorPlugin(schemaFileName string) *scopesDecoratorPlugin {
```

Comment:
As in the above plugin, the package name contains this information so it is not needed here.

```suggestion
func NewPlugin(schemaFileName string) *scopesDecoratorPlugin {
```